hoomd/mpcd/CollisionMethod.h

```diff
@@ -103,9 +107,22 @@ class PYBIND11_EXPORT CollisionMethod : public Autotuned
     uint64_t m_period;        //!< Number of timesteps between collisions
     uint64_t m_next_timestep; //!< Timestep next collision should be performed
 
+    bool m_checked_collision_warnings; //!< True if collision related warnings have been checked
+
+    GPUArray<Scalar4> m_initial_velocity; //!< Initial velocities of the embedded particles
```

Comment:
We may need to change the name of this to something more descriptive, but you can leave it as is for now until we have everything fleshed out.